proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -310,6 +312,8 @@ message PayloadPluginDeleted {
 
 message PayloadUserCreated {}
 
+message PayloadUserActivated {}
```

Comment:
Perhaps this should be `UserReactivated`?

Reply:
hm yeah I agree that would be more accurate. Let me change